Automattic/jetpack
# My Jetpack: Add 'unlinked' parameter to the checkout URL when the user is not connected

Fixes https://github.com/Automattic/jetpack/issues/22731

#### Changes proposed in this Pull Request:

* My Jetpack: Adds `unlinked` parameter to the checkout URL when the user is not connected

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to My Jetpack
* Ensure the user is not connected to Jetpack
* Go to the add-search route
* Confirm, once the user clicks on the Add button, before to gets the Checkout page, it will try to connect with Jetpack.
`https://<your-testing-site>/wp-admin/admin.php?page=my-jetpack#/add-search`

https://user-images.githubusercontent.com/77539/153032585-fab310a0-9cc9-4440-aa9e-f7e015ef2c5f.mov
